config/environments/test.rb

```diff
@@ -1,3 +1,5 @@
+require "active_support/core_ext/integer/time"
```

Comment:
You added this require here. Based on [this answer](https://stackoverflow.com/a/67649699/), I suspect it coming from new Rails templates themselves. However, you only included this here, while also `development.rb` and `docker_development.rb` contain a time-related statement: `2.days.to_i`.

Reply:
Thats strange. I also guess it came from the new templates. Do we remove it here or add it in the other files as well?